services/dy-modeling/client/source/class/qxapp/component/ThreeView.js

```diff
@@ -307,6 +290,30 @@ qx.Class.define("qxapp.component.ThreeView", {
       this._render();
     },
 
+    centerCameraToBB: function() {
+      let center = {
+        x: 0,
+        y: 0,
+        z: 0
+      };
+      if (this.__entities.length > 0) {
+        let unionBBox = null;
+        for (let i = 0; i < this.__entities.length; i++) {
+          const ent = this.__entities[i];
+          if (ent.Name === "PlaneForSnapping") {
+            continue;
+          }
+          const bBox = this.__threeWrapper.getBBox(ent);
+          if (unionBBox === null) {
+            unionBBox = this.__threeWrapper.getBBox(ent);
```

Comment:
= bBox ?

Reply:
hihi I mean it repeats ... you could just write = bBox ...

Reply:
👍